### PR TITLE
Move Xerces lock to Utilities/Xerces

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

This removes an unnecessary external dependency on FWCore/Concurrency.

#### PR validation:

All the code compiles.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
+1

**Comment by Dr15Jones** [comment]
unit test failure is not related to this pull request. It says that the voms proxy was not initialized.

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass (a part the UnitTest which is also failing in the IB)

**Comment by jfernan2** [comment]
+1

**Comment by cvuosalo** [comment]
+1

**Comment by fabiocos** [comment]
@christopheralanwest @ggovi @rekovic @efeyazgan @qliphy @santocch this looks technical and straightforward to me, I would like to integrate it quickly unless you have remarks. Please have a look

**Comment by qliphy** [comment]
+1

**Comment by christopheralanwest** [comment]
+1

File: CondFormats/CTPPSReadoutObjects/BuildFile.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <use name="FWCore/Framework"/>
 <use name="FWCore/MessageLogger"/>
-<use name="FWCore/Concurrency"/>
+<use name="Utilities/Xerces"/>
```
Comment (fabiocos) [inline CondFormats/CTPPSReadoutObjects/BuildFile.xml]: @Dr15Jones @ggovi @smuzaffar where is this really used in this package? This questions goes beyond this PR, that can be merged as far as I am concerned

Comment (fabiocos) [reply CondFormats/CTPPSReadoutObjects/BuildFile.xml]: never mind, I see it, the code was already using Utilities/Xerces https://cmssdt.cern.ch/lxr/source/CondFormats/CTPPSReadoutObjects/src/CTPPSRPAlignmentCorrectionsMethods.cc#0023

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Comment by santocch** [comment]
+1

**Pull request merged**